adminSiteClient/EditorColorScaleSection.tsx

```diff
@@ -60,13 +65,16 @@ export class EditorColorScaleSection extends React.Component<{
 class ColorLegendSection extends React.Component<{
     scale: ColorScale
     features: EditorColorScaleSectionFeatures
+    onChange?: () => void
 }> {
     @action.bound onEqualSizeBins(isEqual: boolean) {
         this.props.scale.config.equalSizeBins = isEqual ? true : undefined
+        this.props.onChange ? this.props.onChange() : noop()
```

Comment:
This might be nicer:

```suggestion
        this.props.onChange?.()
```

Reply:
Oh neat, I didn't know that the null coalescing op could be used that way - thanks!